components/brave_wallet_ui/components/asset-name-and-icon/index.tsx

```diff
@@ -25,7 +25,11 @@ export const AssetNameAndIcon = (props: Props) => {
   return (
     <StyledWrapper>
       <AssetIcon
-        src={assetLogo}
+        // This usage is good, but it's breaking because it looks like it's being loaded from chrome-untrusted://market-display frame
+        // the image is coming from https://assets.cgproxy.brave.com
+        src={`chrome-untrusted://image?url=${encodeURIComponent(
```

Comment:
If you use `createSanitizedImageUrl` from `components/common/create_sanitized_image_url.ts` to create this Url then it will also work in storybook